includes/wikia/services/HubService.class.php

```diff
@@ -105,36 +110,35 @@ public static function getCategoryInfoForCity($cityId) {
 	 */
 	public static function getCategoryInfoForCurrentPage() {
 		$cityId = F::app()->wg->CityId;
+		$categoryId = self::getCategoryIdForCity( $cityId );
 
-		$categoryId = self::getCategoryIdForCity($cityId);
-
-		return self::constructCategoryInfoFromCategoryId($categoryId);
+		return self::constructCategoryInfoFromCategoryId( $categoryId );
 	}
 
 	/**
 	 * Get category id for given cityId
 	 *
-	 * @param int $city_id city id
+	 * @param integer $cityId
 	 *
 	 * @return stdClass ($row->cat_id $row->cat_name)
 	 */
-	private static function getCategoryIdForCity($cityId) {
+	private static function getCategoryIdForCity( $cityId ) {
 		$categoryId = null;
 
-		$category = WikiFactory::getCategory($cityId);
-		if ($category) {
+		$category = WikiFactory::getCategory( $cityId );
+		if ( $category ) {
```

Comment:
use statement `WikiFactory::getCategory( $cityId )` directly in condition instead of assigning it to var above